### DISCUSSION TITLE
Can't ping external networks in the "srsRAN gNB with srsUE" ZMQ scenario.

### DISCUSSION THREAD

**Discussion by Ali78k** [srsran/srsRAN_Project#166] [General Help] [answered]
I've followed the instruction in "[srsRAN gNB with srsUE](https://docs.srsran.com/projects/project/en/latest/tutorials/source/srsUE/source/index.html)" page with _ZMQ option_ and _Open5Gs Core_. Ping and Iperf tests that mentioned were passed but when I want to ping an external network, I get following error:
_Network is unreachable_
![image](https://github.com/srsran/srsRAN_Project/assets/101015446/d620ef71-562f-4c2d-9af5-6cb923f21e0e)

Also, I've attached the Wireshark captured file.
[srsgnb-open5gs-srsue.zip](https://github.com/srsran/srsRAN_Project/files/12319513/srsgnb-open5gs-srsue.zip)

Thanks for your time.

**Comment by pgawlowicz** [comment]
You need to enable IP forwarding - see section "Adding a route for the UE to have WAN connectivity" here:
https://open5gs.org/open5gs/docs/guide/01-quickstart/

**Reply by Ali78k** [reply]
Thank you @pgawlowicz.
I'm sorry I didn't mention this before, but I had enabled IP forwarding according to the link you mentioned.
I also have this problem with Dockerized version in [this link](https://docs.srsran.com/projects/project/en/latest/tutorials/source/flexric/source/index.html).
Isn't it UPF problem? How can I check it?
Thanks a lot.

**Reply by pgawlowicz** [reply]
Did you configure routing correctly? UE needs to have a default route.

**Comment by pgawlowicz** [comment, marked as answer]
Please check the updated tutorial on connecting srsUE with gnb. There is a new part explaining how to configure routes for the PC running the core network and for the UE.

**Reply by joelrenjith** [reply]
Hey @pgawlowicz ,
Do you mind sharing the link of the new tutorial?

**Reply by pgawlowicz** [reply]
https://docs.srsran.com/projects/project/en/latest/tutorials/source/srsUE/source/index.html#ue-ip-forwarding
